lib/ansible/modules/windows/win_unzip.py

```diff
@@ -103,18 +95,43 @@
         src: C:\Downloads\ApplicationLogs.zip
         dest: C:\Application\Logs
         recurse: yes
-        rm: true
+        delete_archive: yes
 
 # Install PSCX to use for extracting a gz file
 - name: Grab PSCX msi
   win_get_url:
     url: http://download-codeplex.sec.s-msft.com/Download/Release?ProjectName=pscx&DownloadId=923562&FileTime=130585918034470000&Build=20959
-    dest: C:\pscx.msi
+    dest: C:\Windows\Temp\pscx.msi
+
 - name: Install PSCX
   win_msi:
-    path: C:\pscx.msi
-- name: Unzip gz log
-  win_unzip:
-    src: C:\Logs\application-error-logs.gz
-    dest: C:\ExtractedLogs\application-error-logs
+    path: C:\Windows\Temp\pscx.msi
+'''
+
+RETURN = r'''
```

Comment:
None of the modules describe `skipped` as it can be caused by the task executor, I guess.